src/Modules/OrchardCore.Commerce/Startup.cs

```diff
@@ -378,11 +385,14 @@ public override void ConfigureServices(IServiceCollection services)
 [RequireFeatures("OrchardCore.ContentLocalization")]
 public class ContentLocalizationStartup : StartupBase
 {
+    // Make sure it's higher than OrchardCore.Commerce.Startup's Order so this loads afterward.
+    public override int Order { get; } = new Startup().Order + 1;
```

Comment:
maybe define constants so you don't have to create an instance of Startup(). We are doing the same thing in OC https://github.com/OrchardCMS/OrchardCore/blob/9a547142e04cf12d7a8597be396a720154e8a842/src/OrchardCore/OrchardCore.Abstractions/OrchardCoreConstants.cs#L32